docfx.json

```diff
@@ -59,6 +59,13 @@
                     "csharp-8.0/nullable-reference-types-specification.md"
                 ]
             },
+            {
+                "files": ["Compiler Breaking Changes - DotNet 7.md"],
```

Comment:
Is there a way to configure this to be published under a different name? The URL will look bad and very inconsistent with all other documents in docs.microsoft.com :(

https://review.docs.microsoft.com/en-us/dotnet/csharp/whats-new/breaking-changes/compiler%20breaking%20changes%20-%20dotnet%207?branch=pr-en-us-28769

Reply:
I didn't find one. I agree, and I'll ask for that feature.